CONTRIBUTING.md

```diff
@@ -211,3 +211,39 @@ command above.
   test outputs are in one file.
   However, it may slow down the tests for not running in parallel
   and may cause the test to timeout.
+
+## Contributing to Keras applications
+
+Contributions to the
+[pre-trained application library](https://keras.io/api/applications/)
+are welcome. Code for Keras applications is located in Keras repository in
+[keras/applications](https://github.com/keras-team/keras/blob/master/keras/applications).
+When contributing to Keras applications, please keep following checklist in
+mind.
+
+ - Keras applications must implement an estabilished and widely used model.
+   Applications should include a link to a paper describing the architecture of
+   the model with at least 20 citations.
+ - Applications should be provided with pre-trained weights.
+   - When submitting a pull request for a Keras application, these weights can
+     be provided at any publically available URL (e.g. a personal Cloud Storage
+     bucket). The weights will be uploaded to a Keras storage bucket while
+     merging the pull request.
+   - Weights should be downloaded with the
+     [get_file](https://keras.io/api/utils/python_utils/#getfile-function)
+     utility function. Be sure to include the `file_hash` argument, which allows
+     cache invalidation on the downloaded weights. The command line programs
+     `shasum` and `sha256sum` can compute a file hash.
+ - You should help us verify that the accuracy of the model with pre-trained
+   weighted matches the reported results of the cited paper.
+ - You should add any new applications to the unit tests defined in
+   `applications_test.py` and `applications_load_weight_test.py`.
+ - Avoid putting computation in the `preprocess_input` function. For new
+   applications `preprocess_input` should pass through user inputs unaltered.
+   Prefer writing application that can handle raw inputs directly.
+   [Preprocessing layers](https://keras.io/guides/preprocessing_layers/) may
+   help with this.
+ - You shoud include a companion pull request to the keras.io
```

Comment:
"Once the PR is approved, you should create" (no need to update keras.io until we're actually publishing the new model)